.github/workflows/release.yaml

```
@@ -57,3 +57,51 @@ jobs:
       run: |
         make release
         gh release upload ${TAG} release.yaml
+
+    - name: Update docs after release creation
+      env:
+        PREVIOUS_TAG: ${{ github.event.inputs.tags }}
+        NEW_TAG: ${{ github.event.inputs.release }}
+      run: |
+        # Update README.md with new tag
+
+        sed -i 's#https://github.com/shipwright-io/build/releases/download/'"$PREVIOUS_TAG"'/release.yaml#https://github.com/shipwright-io/build/releases/download/'"$NEW_TAG"'/release.yaml#g' README.md
+        sed -i '/Examples @ HEAD/a | ['"$NEW_TAG"'](https://github.com/shipwright-io/build/releases/tag/'"$NEW_TAG"')    | [Docs @ '"$NEW_TAG"'](https://github.com/shipwright-io/build/tree/'"$NEW_TAG"'/docs) | [Examples @ '"$NEW_TAG"'](https://github.com/shipwright-io/build/tree/'"$NEW_TAG"'/samples) |' README.md
+
+    - name: Create Readme commits
+      run: |
+        git config user.name ${{ github.actor }}
+        git config user.email ${{ github.actor }}@users.noreply.github.com
+        git add README.md
+        git commit -m "Update Readme with new Tag ${{ github.event.inputs.release }}"
+        git clean -f
+    - name: Create Readme PR
+      uses: peter-evans/create-pull-request@v3
```

This uses an action from an individual's account where we agreed to use the SHA. Would then likely be:

```suggestion
      uses: peter-evans/create-pull-request@9825ae65b1cb54b543b938503728b432a0176d29
```

On the other hand, @HeavyWombat would probably have the single GitHub CLI command at hand that performs the very same change without the need of an external action. I'd prefer the CLI command, but the action is also fine with me.